### PR TITLE
feat(HQM-195): Show and copy bech32 address

### PR DESCRIPTION
**Pull request by devkudasov**
Fixes #

## Proposed Changes

- Copy menu for addresses
- Show bech32 at receive screen
Screenshot 2023-10-19 at 12 28 11
Screenshot 2023-10-19 at 12 28 25
Screenshot 2023-10-19 at 12 28 39

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -117,6 +117,7 @@
     "react-native-mmkv": "2.10.2",
     "react-native-os": "https://github.com/aprock/react-native-os#938676ebd92d58b8928168a7e78cbe216838ed17",
     "react-native-permissions": "^3.10.0",
+    "react-native-popup-menu": "^0.16.1",
```
Comment (ragozin-nikita) [inline package.json:120]: Please fix the version and recreate lock file
```suggestion
    "react-native-popup-menu": "0.16.1",
```

File: src/components/settings-account-detail/settings-account-detail.tsx [hunk 2]
```diff
@@ -83,7 +85,16 @@ export const SettingsAccountDetail = ({
...
           {wallet.name}
         </Text>
-        <Text t14>{wallet?.address}</Text>
+        <AddressInfo copyValue={wallet?.address}>
+          <Text t14>{wallet?.address}</Text>
+        </AddressInfo>
+        <View style={styles.hDevider} />
+        <AddressInfo copyValue={Cosmos.addressToBech32(wallet?.address)}>
+          <Text t14 color={Color.textBase2}>
+            Bech32:{' '}
```
Comment (ragozin-nikita) [inline src/components/settings-account-detail/settings-account-detail.tsx:94]: Please add as i18n value

File: src/components/account-detail.tsx [hunk 3]
```diff
@@ -20,8 +24,14 @@ import {createTheme} from '@app/helpers';
 import {I18N} from '@app/i18n';
 import {Wallet} from '@app/models/wallet';
 import {sendNotification} from '@app/services';
+import {Cosmos} from '@app/services/cosmos';
 import {GRADIENT_END, GRADIENT_START} from '@app/variables/common';
 
+enum TabNames {
+  evm = 'evm',
```
Comment (ragozin-nikita) [inline src/components/account-detail.tsx:31]: You can use getText(I18n.evmTitle) here as a value

**Review by ragozin-nikita — CHANGES_REQUESTED** [review]

File: src/components/account-detail.tsx (hunk 3 above)
Comment (devkudasov) [reply src/components/account-detail.tsx:31]: Type  string  is not assignable to type  number  as required for computed enum member values.

File: package.json (hunk 1 above)
Comment (devkudasov) [reply package.json]: Done

File: src/components/settings-account-detail/settings-account-detail.tsx (hunk 2 above)
Comment (devkudasov) [reply src/components/settings-account-detail/settings-account-detail.tsx]: Done

File: src/components/account-detail.tsx (hunk 3 above)
Comment (ragozin-nikita) [reply src/components/account-detail.tsx:31]: ts bug btw: https://github.com/microsoft/TypeScript/issues/40793

Comment (devkudasov) [reply src/components/account-detail.tsx:31]: Link to issue for future investigation https://github.com/microsoft/TypeScript/issues/40793

**Pull request merged**